assets/src/blocks/widgets/event-attendees.js

```diff
@@ -0,0 +1,46 @@
+const {__}                = wp.i18n; // Import __() from wp.i18n
```

Comment:
This should be `import { __ } from '@wordpress/i18n'`  because we import from the package now.